feat: new rule `tag-line`; fixes #93

brettz9 opened the pull request:
Includes a `always` vs. `never` option as well as two special options for `always` (`count` and `noEndLine`). Fixes #93.

golopot commented:
What does `noEndLine: true` do? I thought it bans empty lines after last tag but there is no tests for it?

brettz9 commented:
There are tests for it, but I can look to add some passing tests instead of only failing ones.  You might want to hold off a little, as I think I have some fixing to do as far as line numbers (and am seeking to add tests for that).

golopot commented:
Do you think it would be better if the always/never options is dropped, and that fee "count: 0" means no empty lines are allowed?

brettz9 commented:
> Do you think it would be better if the always/never options is dropped, and that fee "count: 0" means no empty lines are allowed?

The one reason I thought it may be ok, as is, is because we can have a sensible defaults for "always" which avoids the need for specifying a number in that case as with "never". But I'm ok with switching it if you think it should be changed.

I've updated, btw, so the line number tests/fixes and added test should otherwise be ready for review.

golopot commented on .README/rules/tag-lines.md:
```diff
@@ -0,0 +1,28 @@
+
+##### `noEndLine` (defaults to `false`)
+
+Use with "always" to indicate tag lines should not be added at the end.
```
I don't understand this option. What is a tag line?

brettz9 replied:
Clarified to state:

> Use with "always" to indicate the normal lines after tags should not be
added after the final tag.

Also changed `noEndLine` to `noEndLines` as can be more than one between tags normally.

gajus commented:
:tada: This PR is included in version 34.4.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/eslint-plugin-jsdoc/v/34.4.0)
- [GitHub release](https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v34.4.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
